### ISSUE TITLE
ncm-spma: el7 yum has fastmirror enabled by default

### ISSUE THREAD

**Issue by stdweird** [quattor/configuration-modules-core#525]
and it will use (external) mirrors if it can.
we need a way to disable it, e.g. if a proxy is defined.

**Comment by stdweird** [comment]
also EL7 sets up default yum cron jobs that download rpms and things like that. should also be controlled by spma.

**Comment by stdweird** [comment]
on the cron jobs: `yum` installs the the cron files,
```
# rpm -q --filesbypkg yum |grep cron
yum                       /etc/cron.daily
yum                       /etc/cron.daily/0yum-daily.cron
yum                       /etc/cron.hourly
yum                       /etc/cron.hourly/0yum-hourly.cron
yum                       /etc/yum/yum-cron-hourly.conf
yum                       /etc/yum/yum-cron.conf
```

however, the actual binary that does the work is part of `yum-cron`, so if that  is not installed, no issues with cron doing yum stuff.

**Comment by stdweird** [comment]
fixed by #535
